hs_core/templatetags/hydroshare_tags.py

```diff
@@ -88,3 +90,7 @@ def best_name(content):
         content = content.username
 
     return content
+
+@register.filter(expects_localtime=True)
+def parse_iso(value):
+    return datetime.datetime.strptime(value, "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
This is also a filter that will be used for range slider. I just removed it for now.